thaw/src/avatar/mod.rs

```diff
@@ -85,10 +86,10 @@ pub fn Avatar(
                 }
             }}
             {move || {
-                let src = src.with_value(|s| s.get());
+                let src = src.with_value(|s| if image_failed.get() { None } else { s.get() });
```

Comment:
After the first image loading fails, modify the src, and the image will never be displayed.

tips: `MaybeProp` has implemented Copy, and the `src` prop can remove the `StoredValue` wrapper.